.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,17 @@
-# Pull Request Template
-
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+# Pull Request
 
 Fixes # (issue)
 
+Please ensure that the title of the Pull Request accurately summarizes the intent of the Pull Request. "Adds question" is too generic. "Adds CSS Grids question" or "Adds JavaScript question about fat arrow syntax" is preferred.
+
+Please include relevant motivation and context when adding questions to this project. Including an anticipated answer to the question(s) is appreciated and reduces the need for further discussion.
```

Comment:
See #300 #11 and contributing.md

Reply:
I understand that this repo is intended to be the resource of questions and not answers. This is not intended for the author to give an exhaustive clinical description of the question or even add any answers to the README file.  

That being said, the intent of this was to instruct the author of the pull request to at least provide context of the question. There are plenty of cases in this repo's history where the maintainers are asking what an expected answer is to be and the motivation behind the question.